### PR TITLE
Make the demo notebook more visible in the README

### PR DESCRIPTION
**Pull request by ArthurConmy**

### CODE REVIEW

**Comment by ArthurConmy** [comment]
Sadly, I'm seeing this run out of RAM even on V100s for all setups I've tried so far

**Comment by alan-cooney** [comment]
> Sadly, I'm seeing this run out of RAM even on V100s for all setups I've tried so far

Are you running out of memory on train or validation?

The demos use the Anthropic Towards Monosemanticity defaults, which include a very large batch size (8k). Coupled with the fact that we're training there on a larger residual stream (d_model=512), it is pretty resource hungry so I'm not surprised a single V100 (esp if 16GB) can't take it. We can add a note for a suggestion to reduce the batch size - or just put a default size that is a bit smaller (but we'll need to do a new sweep to find sensible defaults if so for l1 coefficient).

File: README.md [hunk 1]
```diff
@@ -10,6 +10,12 @@ A sparse autoencoder for mechanistic interpretability research.
 [![Read the Docs
 Here](https://img.shields.io/badge/-Read%20the%20Docs%20Here-blue?style=for-the-badge&logo=Read-the-Docs&logoColor=white&link=https://ai-safety-foundation.github.io/sparse_autoencoder/)](https://ai-safety-foundation.github.io/sparse_autoencoder/)
 
+Train a Sparse Autoencoder in colab: <a target="_blank" href="https://colab.research.google.com/github/ai-safety-foundation/sparse_autoencoder/blob/main/docs/content/demo.ipynb">
+  <img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/>
+</a>
+
+Or install for your project:
```
Comment (alan-cooney) [inline README.md:17]:
```suggestion
Train a Sparse Autoencoder [in colab](https://colab.research.google.com/github/ai-safety-foundation/sparse_autoencoder/blob/main/docs/content/demo.ipynb), or install for your project:
```

To make it look a little better

Comment (alan-cooney) [reply README.md:17]: Esp as most people actually probably want to install with a beefier machine than the colab runtimes, if they intend on training some saes

**Review by alan-cooney — CHANGES_REQUESTED** [review]

**Review by alan-cooney — APPROVED** [review]

**Pull request merged**